app/util.py

```diff
@@ -37,7 +37,22 @@ def loads(response: httpx.Response) -> Any:
     return orjson.loads(response.text)
 
 
-def commit_and_refresh(session: Session, instance: T) -> T:
+class SortOrder(StrEnum):
+    ASC = "asc"
+    DESC = "desc"
+
+
+def get_order_by(sort_field: str, sort_order: str, model: type[SQLModel] | None = None) -> Any:
+    if "." in sort_field:
+        model_name, field_name = sort_field.split(".", 1)
+        # credere-frontend doesn't use any camelcase models.
+        column = getattr(getattr(models, model_name.capitalize()), field_name)
+    else:
+        column = getattr(model, sort_field)
+    return getattr(col(column), sort_order)()
+
+
+def commit_and_refresh(session, instance):
```

Comment:
Need to keep annotations from merge conflict

Reply:
Done